.cargo/config.toml

```diff
@@ -1,6 +1,7 @@
 [build]
 rustdoc = "rustdoc"
 rustdocflags = ["--show-type-layout", "-Z", "unstable-options"]
+rustflags = ["-L", "/opt/homebrew/lib/"]
```

Comment:
Surely this shouldn't exist here?

Reply:
Ugh yes it really shouldn’t, but for some reason `ld` on mac is refusing to find `zstd` installed on my machine. This is the only way I found to fix it. This will be ignored on systems that this doesn’t exist since its only meant to “indicate” where to look for this libraries.

Reply:
True, it won't really hurt others I guess

Reply:
I would love to remove this dumb hack, I will leave a `##Hack`

Reply:
This now magically works - I think I had a borked installation locally. Removed!